app/environment.py

```diff
@@ -35,14 +34,20 @@ def display_env_state(env_state: EnvironmentState):
 
         if env_state.action_id is not None:
             st.write("**:blue[Blue Agent:]**")
-            action = NodeAction.from_id(env=env_state.env, action_id=env_state.action_id)
-            if action.node_property != NodePropertyAction.NONE:
+            try:
+                action = NodeAction.from_id(env=env_state.env, action_id=env_state.action_id)
+            except Exception:
+                pass
+            if (
+                action.node_property != NodePropertyAction.NONE
```

Comment:
getting an error here if the try clause fails because action is uninitialised

Reply:
fixed